portaudio/all/conanfile.py

```diff
@@ -73,7 +73,7 @@ def configure(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename("portaudio", self.sources_folder)
+        os.rename("portaudio-19.7.0", self.sources_folder)
```

Comment:
While this file seems to be trying to be version agnostic, there are other places the version is hardcoded in here so I didn't think this was too egregious.

Reply:
No worries, I'll fix it when we will be pulling 19.8.0 :)